### PR TITLE
This switches off the lxqt-notifications on autostart - (Solves- #133)

### LINKED ISSUE #133: Disable Desktop Notifications
Upon first startup the Meilix pollutes the screen with LXQT notifications. So the question is how to turn them off or silence or tweak them? How to "fix them" in so far they demonstrate an error?

Hints for research:
https://github.com/lxde/lxqt-notificationd
https://wiki.archlinux.org/index.php/Desktop_notifications

### PR DESCRIPTION
**Pull request by pranav1698**
### Short description
This disables the lxqt-notifications on autostart, This suggests only removing the autostart module which causes the desktop to be filled with notifications on startup.
If a user wants to reset the notifications the user can just open the `lxqt-config-notificationd`. 
I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

Fixes #133

### CODE REVIEW

**Comment by abishekvashok** [comment]
We need to check on this as I am not sure whether deleting the module is a good way instead of find a way to disable it.

**Comment by pranav1698** [comment]
@tabesin @abishekvashok I have tested all my changes with the ISO, also I will further test these changes

File: scripts/chroot_reconst.sh [hunk 1]
```diff
@@ -64,6 +64,9 @@ apt-get -qq -y remove xscreensaver
 # This switch offs the sleep mode
 sudo systemctl mask sleep.target suspend.target hibernate.target hybrid-sleep.target
 
+# This switches off notifications(autostart)
+rm /etc/xdg/autostart/lxqt-notifications.desktop
```
Comment (tabesin) [inline scripts/chroot_reconst.sh]: Is there no other way to silence that? 

Anyway, this could be an interim fix for the related stub (dummy feature).

Comment (pranav1698) [reply scripts/chroot_reconst.sh]: I am really struggling in finding an alternative to this, 
I have researched on this and deleting the module on autostart comes out to be the least worst method to me as users will be able to config it again using `lxqt-config-notificationd`

Comment (pranav1698) [reply scripts/chroot_reconst.sh]: I will test more on this

**Comment by tabesin** [comment]
A kill of the notification queue should suffice. That is what we want.

But the truth is that no one knows how to perform this. Tarun might reall more because he researched it. Otherwise LXQT dev should know or maybe we find it in the doc.

**Pull request merged**